### DISCUSSION TITLE
Ant Media server circle not allowing publish

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6394] [Plugins/Custom Extensions]
Ant Media server circle not allowing publishing to rtmp , or adding rtmp end point to publish the  meeting , what is the problem  , please advice

**Comment by brixxnatt** [comment]
anyone ?

**Reply by yashtandon113** [reply]
Hi @brixxnatt 
You cannot directly broadcast the conference meeting to any other RTMP end point.

We recommend you use the Media Push plugin to merge participants into one stream and then restream it to RTMP end points. Check out the below blog post to create the merged conference room stream.

https://antmedia.io/conference-call-recording/

I hope it helps. Please feel free to ask if you have any questions.
